src/pages/ImportData.js

```diff
@@ -58,6 +58,21 @@ const ChoiceButton = ({ iconName, title, detail, style, ...props }) => {
   ])
 }
 
+const ResponseFragment = ({ title, snapshotResponses, responseIndex }) => {
+  const { status, message } = snapshotResponses ? snapshotResponses[responseIndex] : {}
+  const [color, iconKey, children] = Utils.switchCase(status,
```

Comment:
If you make this return children as a `Fragment` and render it with `[]` below, you can just do `children &&` for the condition

Reply:
I pushed a diff update that would shorten the condition too. Is this okay?

Reply:
I'd still like to avoid passing children to `div()` not in an array, so I'd like if it could be a Fragment coming out of the cond. That said, couldn't you just use `!!children` as the condition now? It's never present but empty.